projects/js-packages/charts/src/components/bar-chart/stories/index.docs.mdx

```diff
@@ -276,7 +275,7 @@ By default, bars with zero values have no visual height, making them difficult t
 
 Display series information with automatic color and pattern matching:
 
-<Canvas of={ LegendStories.Default } />
+<Canvas of={ BarChartStories.Default } />
```

Comment:
This canvas isn't displaying a legend

![Screenshot 2025-08-05 at 12 52 27 PM](https://github.com/user-attachments/assets/9a23ef82-237f-4017-a562-1ebc06c264cc)